client/devdocs/design/blocks.jsx

```diff
@@ -115,7 +115,7 @@ export default class AppComponents extends React.Component {
 				<Collection
 					component={ this.props.component }
 					filter={ this.state.filter }
-					section="blocks"
+					section={ 'blocks' }
```

Comment:
Hmm, why is this preferred? Other parts of the patch use `=""` strings rather than `={ '' }` ones.